ADD: Added verification functionality for checking if user is owner

Foopert opened the pull request.

NickLewanowicz commented on db/db.js:
```diff
@@ -55,5 +55,26 @@ const createSection = async (attrs) => {
+    // Todo: validate you are an owner of the project
+    const sectionRef = findRecord(Section, attrs.section)
+    if (sectionRef) {
+        return await saveRecord(Section, attrs.section)
```
'createTask' should be saving a 'Task' model not a 'Section' model. Also you need to pass ALL the attrs not just section id. Should look like this:

return await saveRecord(Task, attrs)

NickLewanowicz commented on graphql/resolvers.js:
```diff
@@ -124,10 +124,14 @@ const resolvers = (models) => ({
+    async createTask(root, args, context) {
+      const userId = getUserId(context)
+      if (await db.userOwnsSection(args.section, userId)) {
+        const task = new models.Task(args);
```
what is inside of this conditional should be moved to the db.createTask, would look like this:
if (await db.userOwnsSection(args.section, userId)) {
    return await db.createTask(attrs)
}

then in createTask you would set the `created` attribute and return the savedRecord

Foopert replied:
attrs isnt defined in resolvers.js, should it be args instead?

NickLewanowicz commented on utils.js:
```diff
@@ -8,7 +8,7 @@ function getUserId(context) {
     return userId
   }
-  return ""
+  return userId
```
REMOVE THIS. If the user isnt authorized this should return empty

Foopert commented:
- Fixed functionality for createTask function, now passes all args and saves record with attrs
- Removed userId in utils.js (big vulnerability, was only used for testing but I missed it, thanks for the catch)
